scalyr_agent/util.py

```diff
@@ -698,6 +698,14 @@ def get_parser_from_config(base_config, attributes, default_parser):
     return default_parser
 
 
+def get_web_url_from_upload_url(server):
```

Comment:
Let's add a unit test for this.  I'd like to check all these cases, using both the US and EU URLs.  Also, add in a test for `logstaging.scalyr.com` to verify it comes through unchanged.